### ISSUE TITLE
The current state of morale needs a change

### ISSUE THREAD

**Issue by UpgrayeDD** [CleverRaven/Cataclysm-DDA#30979]
**Is your feature request related to a problem? Please describe.**  
     I understand that changes to the morale system that were made in order to make it harder to achieve the elated state in game, especially through using multiple bonuses.  However the intended result has been overshot and has made it near impossible to achieve the elated state without the use of hard drugs (ie. crack, heroin..) in game.  Using the sum of squares square rooted for both a single source stacking as well total moral is a bit too heavy handed.
**Describe the solution you'd like** 
      similar to what someone suggested earlier I think this formula would work better
(m1^1.4+m2^1.4+m3^1.4...)^1/1.4 
This better suits the intended goal of making it harder to achieve 100 morale without making it damn near impossible.
**Describe alternatives you've considered**  
-Revert the changes to pre morale overhaul changes
-Make a mod reverting to pre morale overhaul changes
-another formula that doesn't include exponents Ex: (m1+1/2m2+1/2m3+1/2m4...) where m1 is the highest value

**Additional context**  
     The original system allowed for bonus stats all the way up to 200 and if you include to the last two 25 increment speed bonuses 250.  This implies that it was meant for these values to be achievable. Now my proposed change doesn't even ask for those values to be possible(which without any doubt they are currently not possible).  I just think that getting over the 100 mark without becoming a crackhead should at least be something that can be done.  
     Another problem with the current system is that it makes almost all traits having anything to do with positive happiness worthless.   Sure you might be able to get some slight value early but the second you have the ability to have any other source of happiness the benefit the supply become so negligibly small that its depressing.  The reason to take these perks was that it makes the higher numbers more easily accessible.  However in the current system they become a single buckets worth of water trying help fill a swimming pool.
     And the opposite becomes true.  Any trait that adds negatives becomes so much worse when trying to achieve higher values.  The needed value to achieve 100 morale before being square rooted is a whopping 10000 versus what's needed to get 110 to make up for a -10 is 12100.  That means that you need a morale value of (2100)^1/2 or 46 in order to make up for that -10.  That feels very unbalanced to me.  At least with my currently proposed changes it would only require 25 morale to offset the -10 instead of 46.
     Even if we want to look at it from a realism perspective the current system doesn't make the cut.   If I was drink some high quality alcohol, eating my favorite foods, and having the leisure time to enjoy good books and music my mood would be pretty damn good.  Put in a scenic view and that's literally a vacation.  You say that a bunch of little things that add up shouldn't have such a great affect on your mood, but I have to disagree.  It's when all the little things are going your way that you can be in a very positive state of mind.
    I remember someone in the other thread talking about how it should be some really great event that should be what's able to put you into great mood.  But in cataclysm those events are non existent.  The only cdda gives us to fill that hole is hard drugs.  Messaging aside, I just think it's nonsensical that crack and heroin are the only ways to break 100 morale.  I understand the goal of those pushing the morale overhaul forward wanted to make it more difficult to achieve the elated state and would like to ask each of them if they managed to break 100 morale after the change without using crack and/or heroin.
TL/DR I just want to be happy again

**Comment by kevingranade** [comment]
The elated state didn't factor into the change at all, but now that I'm looking at it, I have no problem with it becoming inaccessable.
It's a silly effect, and especially the upper levels of it that double speed or increase stats by one or more points are simply absurd. 

I'm not saying morale handling is exactly where it should be, the change you reference was a very coarse adjustment after years of power creep, but nothing involving elation bonuses presents a compelling case for a change.

**Comment by I-am-Erk** [comment]
I'm with Kevin here. It's likely the first pass change to morale needs adjusting, but your character is living in a darkened wasteland, after losing everyone and everything close to them. "Elated" isn't an emotional state they should be readily reaching. Using it as a core argument is like saying 300lbs is a healthy weight because that's how high the doctor's office scale goes.

**Comment by UpgrayeDD** [comment]
Pertaining to Kevin
If the elated state had no impact on the decisions to overhaul moral I fail to see the reason for even bothering to tinker with it in the first place?  If it had no bearing than what other reason was there for such a drastic change?  You claim that the elated state is silly but human history might have something to say about that.  Morale often plays a of the success of failure of many aspects of life ranging from production quantity and quality to military efficiency.
Happy workers work harder is not just a slogan but a fact of life.  Anyone with any kind of management experience should be able to understand this.  Just looking at the rules placed on this forum we can see that they are meant to ensure friendly environment for discussion.   So as to not alienate the contributors on the project and discouraging further attempts to contribute.  Because if they are unhappy every time they come to contribute they won't come back.
The same can apply on a personal scale as well.  When people are in a good mood or excited they have a spring in their step.  They feel like they can go out to accomplish and do more.  The game has support for if you are in such a terrible mood you just don't want to do anything.  Why is it so terrible for the opposite to be true?
Pertaining to I-am-Erk
I think anyone that is going to survive in that darkened wasteland has to be both adaptable and able to find joy in life/a reason to keep going.  Scraping by with enough food and water isn't enough for most people to want to keep moving forward in an environment like that.  
As far as your scale analogy goes anyone who goes outside in first world nation can see that there is more than enough need for those scales and that it is all too possible.  And the previous scale went to 250 where the changes made it near impossible to break 70's.  We shouldn't be basing the top end of the scales around anorexic school girl either.


Maybe the wording should be changed from "elated" to "motivated", "cheerful" ", or "satisfied".  This might seem more fitting to the setting then the wording of "elated".

**Comment by I-am-Erk** [comment]
I don't think you're following a lot of what's being said here. I'll try to break it down.
- The morale scale probably does need rebalancing.
- The morale scale needed rebalancing to begin with. See the original PR for reasons, why make me retype them again?
- The upper limits of the morale scale are not an indication of where the player should normally sit.
- Being in a good mood after the end of the world while being chased by zombies is difficult.

I have no idea why you're talking about management techniques. Game avatars aren't our employees, we're not adjusting their morale to try to get more efficient labour hours out of them. And the people changing morale systems are contributors as well as anyone else; very active ones, in fact. Not sure why we don't count in the "do not alienate the contributors" thing you're talking about here.

Certainly people able to be happy after the end of the world are likely to do better. There's no implicit guarantee your player is one of those people. The game starts immediately after the world ends, not a year later when the stragglers have been picked off.

**Comment by kevingranade** [comment]
This isn't going anywhere. I'm not interested in making changes to make the related state easier to reach.
